cli.js

```diff
@@ -221,11 +202,4 @@ function init(args, options) {
 sudoBlock();
 updateNotifier({pkg: pkg}).notify();
 
-if (process.stdin.isTTY) {
-	init(args, options);
-} else {
-	getStdin(function (data) {
-		[].push.apply(args, data.trim().split(/\r?\n/));
-		init(args, options);
-	});
-}
+init(subarg(cli.input, options)._, cli.flags);
```

Comment:
The docs say you can, it's the `minimist` options object. But the options object defined by us is only defining "global" arguments, not the arguments that can be passed in per sub command. At least, that's what it looks like to me :). So I think we just might leave it out here or define a second `minimist` object that defines the arguments of the subcommand.